torch/distributed/distributed_c10d.py

```diff
@@ -2219,14 +2219,23 @@ def all_gather(tensor_list, tensor, group=None, async_op=False):
         work.wait()
 
 
-def _all_gather_base(output_tensor, input_tensor, group=None, async_op=False):
+def all_gather_into_tensor(output_tensor, input_tensor, group=None, async_op=False):
     """
-    Single tensor all gather. Gathers a single tensor from all ranks, and puts them in a single output tensor.
+    All ranks gather tensors from all other ranks and put them into a single
```

Comment:
I'm not certain but is the first "ranks" or "All ranks" needed?

referenced what https://pytorch.org/docs/stable/distributed.html?highlight=all_gather#torch.distributed.all_gather says

Reply:
Updated now to "Gather tensors from all ranks and put them into ..."